api/manager-api.service

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[Unit]
+Description=Manager-API service
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=5
+WorkingDirectory=<"Your Directory">/apisix-dashboard/output
```

Comment:
Does the standard release have this directory? @nic-chen

Reply:
Yes, if it is built from source.  The build script takes care of that. However, for rpm tarball, the directory structure is slightly customized.

Reply:
@tokers  After the build, there will be an `output` directory.
The current problem is to choose a suitable library to implement this feature. Please check the mailing list when you have time to discuss it. @bisakhmondal has started a discussion. thanks.

Reply:
do we have created an issue about this when we build an RPM package?